src/lib/cimbar_js/cimbar_js.cpp

```diff
@@ -113,7 +113,7 @@ int cimbare_next_frame()
 	return ++_frameCount;
 }
 
-int cimbare_encode(unsigned char* buffer, unsigned size, int encode_id)
+int cimbare_encode(const unsigned char* buffer, unsigned size, const char* filename, unsigned fnsize, int encode_id)
```

Comment:
The api should communicate that these are const (since they *are*, whether they're using the keyword or not).